helm/templates/service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
This service doesn't define any ports to expose if I'm not mistaken. How would you use / reach this in a deployed scenario other then deploying the front-end in the same space and exposing the front-end rather than the back-end?

Reply:
How would you access that front-end with this? What ip:port would you need?

Reply:
From inside the kubernetis -- `http://gaia.gaia:8080`. From outside -- `http://gaia.k8s.dev:80` (could be overriten by specifying custom `values.yaml` file.

Reply:
Oh, I see. I was mistaken in thinking that helm would actually setup the proper ingress from nginx. But I see that that's not helm's job. So basically, for this to work, your ingress already needs to be configured and available.

Reply:
Here is the configuration: https://github.com/gaia-pipeline/gaia/pull/127/files#diff-2973c2b3b551fb152acc1beda0a289cd

Reply:
Yes, I saw that. But in order for that to work you need it enabled.

Okay, how did you test this? What were your steps? :)

Reply:
1. Install and run minikube
2. `eval $(minikube docker-env)`
3. `make deploy-kube`
4. add `gaia.k8s.dev` to `/etc/hosts`
5. open `http://gaia.k8s.dev`

Reply:
```$ minikube addons list
- addon-manager: enabled
- coredns: disabled
- dashboard: enabled
- default-storageclass: enabled
- efk: disabled
- freshpod: disabled
- heapster: disabled
- ingress: disabled
- kube-dns: enabled
- registry: disabled
- registry-creds: disabled
- storage-provisioner: enabled
```

```$ kubectl get po -n kube-system     
NAME                                    READY     STATUS    RESTARTS   AGE
default-http-backend-66f887685f-bhtt9   1/1       Running   7          5d
default-http-backend-66f887685f-zln8v   1/1       Running   12         5d
kube-addon-manager-minikube             1/1       Running   2          5d
kube-dns-7bb84f958d-r8zg9               3/3       Running   6          5d
kubernetes-dashboard-85565c9d85-9rjz8   1/1       Running   4          5d
nginx-ingress-lb-7qp7w                  1/1       Running   1          5d
storage-provisioner                     1/1       Running   2          5d
tiller-deploy-7cb65f74cb-8d86r          1/1       Running   9          5d```

Reply:
You see, you have `nginx-ingress-lb-7qp7w` is the difference.

Also, I used `dnsmasq` instead of updating my /etc/hosts. But that should be of no difference.

Reply:
Actually, let me re-test this. Something didn't work last time I tried this. The ingress should have been enabled by the template.